apps/taiga-lumbermill/src/dashboards/crypto/components/nft/nft-item/nft-item.component.less

```diff
@@ -0,0 +1,48 @@
+.clicked {
```

Comment:
what is "clicked" ?
clicked on card in list in the parent component?

it's a separate view, a separate avatar, you can use it in an independent context, so let's not use the parent state in the style names in the child